lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
I still don't fully agree with this approach. We could add a method which will init the promises library: `cobol.initPromises()` or so. Only after that, the module will return a promise object, otherwise just like before (probably `undefined`).

Would it be possible?

On the other side, I see the potential it comes with. Nice work! :+1: